File: SAM Issue 3/updated SAM files/Java/Q01a.java
public class Q01a 
{
    public static void main(String[] args) 
	{
        int[] myNumbers = {20, 30, 40, 50};
        int total = 0;
        for(int i=5; i<15; i++)
		{
            total = total + i;
            if (i < 10)
                System.out.println(i * 10 );
            else
                System.out.println(i);
        }
        System.out.println(total);
    }
}

File: SAM Issue 3/updated SAM files/C#/Q01a.cs
using System;
namespace Q01a
{
    class Q01a
    {
        static void Main(string[] args)
        {
            int[] myNumbers = { 20, 30, 40, 50 };
            int total = 0;
			for (int i = 5; i < 15; i++)
            {
                total = total + i;
				if (i < 10)
                    Console.WriteLine(i * 10);
                else
                    Console.WriteLine(i);
            }
            Console.WriteLine(total);
            Console.ReadKey();
        }
    }
}